src/Dofs/DirichletBoundaryConditions.jl

```diff
@@ -243,4 +244,4 @@ function meandiag(K::AbstractMatrix)
         z += K[i,i]
     end
     return z / size(K,1)
```

Comment:
Seems to work just fine actually. Since we insert 0 i don't think the sparse structure changes.